packages/openssl/pre_packaging

```diff
@@ -0,0 +1,2 @@
+# abort script on any command that exits with a non zero value
+set -e
```

Comment:
Full list of packages in the stemcell is in the stemcell_dpkg_l.txt file archive. 
See https://github.com/cloudfoundry/bosh/issues/1288#issuecomment-223883624

Additionally, some stemcell have specs in git listing expected packages https://github.com/cloudfoundry/bosh/blob/cc49a8ee67434b3ac89deae81724403283c51bfa/bosh-stemcell/spec/os_image/rhel_7_spec.rb#L57-L58 however I did not find them in the bosh repo for ubuntu.